index.html

```diff
@@ -123,6 +123,21 @@ <h2>Security and Privacy Considerations</h2>
       connects to in the future. Therefore, the user agent MUST obtain consent for
       this feature.
     </p>
+  
+    <p>
+      This feature collects personalized information from websites including their
+      current identity (optional) and media recommendations. This sensitive data
+      should only be stored locally in the user's profile. 
```

Comment:
Should we say "will only be stored" instead of "should only be stored"? We are using "will" below.